maestro-cli/src/main/java/maestro/cli/studio/DevicesService.kt

```diff
@@ -0,0 +1,74 @@
+package maestro.cli.studio
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.response.respondText
+import io.ktor.server.routing.Routing
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import maestro.cli.device.Device
+import maestro.cli.device.DeviceService
+import maestro.cli.session.MaestroSessionManager
+
+private data class SelectDeviceRequest(
+    val instanceId: String?,
+    val modelId: String?,
+)
+
+object DevicesService {
+    private var selectedDevice: Device.Connected? = null
+
+    fun routes(routing: Routing) {
```

Comment:
The choice is entirely up to you, but just FYI one convenient way of declaring such routes is by using Kotlin's extension functions:

```
fun Route.deviceRoutes() {
    devicesRoute()
    getSelectedDeviceRoute()
    selectDeviceRoute()
}

fun Route.devicesRoute() {
    get("/api/devices") {
        // etc
    } 
}

// etc.
```

Then you can mount all the routes like that:

```
routing {
    DeviceService.deviceRoutes()
}
```

Reply:
Ah, neat suggestion! However, won't it be 

```kotlin
routing {
   this.devicesRoutes()
}
```
instead given that it is an extension function for `Route`?